dali/python/nvidia/dali/_utils/external_source_impl.py

```diff
@@ -271,17 +278,19 @@ def get_callback_from_source(source, cycle, batch_info=False):
                 # in the error message.
                 iterator = iter(source)
             iterable = True
-            callback = lambda: next(iterator)
+            callback = lambda: next(iterator) # noqa E731
```

Comment:
I guess it would complain about spaces...?
```suggestion
            callback = lambda: next(iterator)  # noqa E731
```